docs/04-api-reference/front-commerce-core/defineExtension.mdx

```diff
@@ -144,3 +144,14 @@ Example:
   }
 }
 ```
+
+#### `onServerServicesInit`
```

Comment:
**issue:** I haven't detected when `onClientServicesInit` was added, but it should be removed (a few lines above - I can't annotate it directly)

Reply:
It has not yet been added, it's in this MR: https://github.com/front-commerce/developers.front-commerce.com/pull/818

Reply:
Oh right! I commented on the other MR.